scala-di-bench/app/controllers/BenchController.scala

```diff
@@ -52,8 +71,19 @@ class BenchController @Inject() (
     )
   }
 
-  def upload = action(parsers.raw) { request =>
-    Ok("upload")
+  def simpleForm = action { implicit request =>
+    userForm.bindFromRequest.fold(
+      formWithErrors => BadRequest("This should not happen"),
+      user => Ok("It works!")
+    )
+  }
+
+  def upload = action(parsers.multipartFormData) { request =>
```

Comment:
It would be good to have two upload tests, one test for multipart file upload performance and another test for which is just a PUT for raw upload streaming performance without any `BodyParser` overhead.